ksql-engine/src/main/java/io/confluent/ksql/planner/plan/KsqlStructuredDataOutputNode.java

```diff
@@ -49,7 +45,7 @@
   private final KsqlTopic ksqlTopic;
   private final KeyField keyField;
   private final boolean doCreateInto;
-  private final Map<String, Object> outputProperties;
+  private final boolean partitionBy;
 
   @JsonCreator
```

Comment:
can we remove the jackson annotations from this class? it seems like we don't use it and every time it makes me nervous for backwards compatibility

Reply:
Yeah, I'm 99.9% sure they ain't being used... but that's another PR and one I never seem to find time to do,.